wire-ios/Wire-iOS/Sources/UserInterface/Conversation/ConversationRootViewController.swift

```diff
@@ -100,14 +102,15 @@ final class ConversationRootViewController: UIViewController {
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        navigationController?.setNavigationBarHidden(true, animated: animated)
+        navigationController?.setNavigationBarHidden(false, animated: animated)
```

Comment:
`false` is the default, right? Better we'd remove the line.

Reply:
Yep! Removed it!